From 171c76a299f69fab6df460bae30c829743c48af5 Mon Sep 17 00:00:00 2001
From: Nick Hollinghurst <nick.hollinghurst@raspberrypi.com>
Date: Mon, 4 Mar 2024 16:59:54 +0000
Subject: [PATCH] media: i2c: imx296: Updated register setting to fix Fast
 Trigger

In Fast Trigger mode (external shutter control), FE packet was
not sent at end of frame. Sony recommend this change to fix it.

Signed-off-by: Nick Hollinghurst <nick.hollinghurst@raspberrypi.com>
---
 drivers/media/i2c/imx296.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/media/i2c/imx296.c
+++ b/drivers/media/i2c/imx296.c
@@ -472,7 +472,7 @@ static const struct {
 	{ IMX296_REG_8BIT(0x30a4), 0x5f },
 	{ IMX296_REG_8BIT(0x30a8), 0x91 },
 	{ IMX296_REG_8BIT(0x30ac), 0x28 },
-	{ IMX296_REG_8BIT(0x30af), 0x09 },
+	{ IMX296_REG_8BIT(0x30af), 0x0b },
 	{ IMX296_REG_8BIT(0x30df), 0x00 },
 	{ IMX296_REG_8BIT(0x3165), 0x00 },
 	{ IMX296_REG_8BIT(0x3169), 0x10 },
